src/rdkafka_admin.c

```diff
@@ -495,7 +507,13 @@ rd_kafka_admin_coord_request(rd_kafka_broker_t *rkb,
             rkb, &rko->rko_u.admin_request.args,
             &rko->rko_u.admin_request.options, errstr, sizeof(errstr), replyq,
             rd_kafka_admin_handle_response, eonce);
-        if (err) {
+
+        rd_bool_t is_consumer_group_response =
+            rko->rko_u.admin_request.cbs->request ==
+            rd_kafka_admin_ConsumerGroupDescribeRequest;
+
+        if (err && (!is_consumer_group_response ||
+                    err != RD_KAFKA_RESP_ERR__UNSUPPORTED_FEATURE)) {
                 rd_kafka_enq_once_del_source(eonce, "coordinator response");
```

Comment:
We should keep this one generic, what I was suggesting is that we remove `rd_kafka_enq_once_del_source(eonce, "coordinator response");` here as it's causes a bug in these cases when we return without sending the request.
Let's discuss to validate this fix.
```suggestion
        if (err) {
```